server/src/tests/hardcodesampleauth.ts

```diff
@@ -0,0 +1,49 @@
+import fastify from "fastify";
+import { clerkPlugin, getAuth } from "@clerk/fastify";
+
+const CLERK_SECRET_KEY = "sk_test_123"; // Replace with your actual Clerk secret key
+const CLERK_PUBLISHABLE_KEY = "pk_test_123"; // Replace with your actual Clerk publishable key
+
+console.log("Using hardcoded Clerk keys:");
```

Comment:
These must be removed, not sure why you didn't remove unneccessary console.log statements. Even if they aren't your actual keys, it's bad practice to have some dangling statement.